### ISSUE TITLE
can this lead to a block or ban?

### ISSUE THREAD

**Issue by torian257x** [brutalsavage/facebook-post-scraper#52]
it seems fairly difficult creating a dummy facebook account

will this program cause blocking or banning of the facebook account? any good approach to get a dummy account?

**Comment by brutalsavage** [comment]
Yes I would becareful as it can potentially lead to a banned facebook account. I personally have never had this happened but it could be a possibility
